tests/typechecker/field_access_on_enum.jakt

```diff
@@ -1,5 +1,5 @@
 /// Expect:
-/// - error: "Member field access on value of non-struct type ‘E’"
+/// - Error: "Member field access on value of non-struct type ‘E’"
```

Comment:
you'll want to leave this as `error` as it's part of the unit testing framework

Reply:
Oh I committed this change by mistake. Fixing it.